ccm_web/server/src/main.ts

```diff
@@ -10,6 +10,7 @@ import { ConfigService } from '@nestjs/config'
 import { NestFactory } from '@nestjs/core'
 import { NestExpressApplication } from '@nestjs/platform-express'
 import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger'
+import { urlencoded, json } from 'express';
```

Comment:
Codacy has flagged 4 issues that probably also show up in your IDE if you have linting set up. We're using `standard`, so "no unnecessary semicolons".

Reply:
Maybe consider moving this up above `import session from 'express-session'`?

Reply:
Done: removed the semicolons and moved up the line.